cli/lib/kontena/command.rb

```diff
@@ -218,15 +218,19 @@ def run(arguments)
     end
   rescue Kontena::Errors::StandardError => ex
     raise ex if ENV['DEBUG']
-    abort(" [#{Kontena.pastel.red('error')}] #{ex.class.name} : #{ex.message}")
+    Kontena.logger.error("ERROR") { "#{ex.class.name} : #{ex.message}" }
```

Comment:
All backtracelogging biz now done with `logger.error(ex)`. This only works with the default Logger::Formatter which is super noisy with PID's and what not.